.github/actions/test-cli/action.yml

```
@@ -0,0 +1,19 @@
+name: Test basic CLI usability
+inputs:
+  artefact_name:
+    description: Name of the uploaded CLI artefact
+    required: true
+runs:
+  using: "composite"
+  steps:
+    - uses: actions/download-artifact@v4
+      with:
+        name: ${{ inputs.artefact_name }}
+    - name: Test
+      shell: sh
+      run: |
+        chmod +x ./jstz
+        export PATH=$PATH:$PWD
+        jstz account create test
+        jstz account list
+        jstz login test
```

I guess this will have to change a bit because of XDG_CONFIG_HOME change. It might be good to also test that these commands are not silently failing, by for example checking some condition on the prefix of the last output or something similar. 

Otherwise it looks good to me.

---

What does `XDG_CONFIG_HOME` have to do with this? The commands will not be able to fail silently because any non-zero error code fails the step.

---

ah yes, I mistakenly read is modifying permissions of /.jstz folder instead of ./jstz.

Not sure though if currently in the CLI it always returns non-zero code or some of the errors can get suppressed in which case checking the output might be worth doing here.

---

I see what you mean. For now any error captured leads to a non-zero exit code. The main point of this check is to make sure that the executable can be executed. What happened previously was that the executable was not packaged properly and failed when it tried to load some dynamic libraries. This check ensures that such situation does not happen. We should be able to capture other errors in integration tests.